testing/conftest.py

```diff
@@ -5,58 +5,78 @@
 import pytest
 from selenium import webdriver
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
-from wait_for import wait_for_decorator
+from wait_for import wait_for
 from widgetastic.browser import Browser
 
 
+CAPABILITIES = {
+    "firefox": DesiredCapabilities.FIREFOX,
+    "chrome": {
+        **DesiredCapabilities.CHROME,
+        "chromeOptions": {"args": ["disable-dev-shm-usage", "no-sandbox"]},
+    },
+}
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--browser-name",
+        help="name of the browser",
+        choices=("firefox", "chrome"),
+        default="firefox",
+    )
+
+    parser.addoption("--force-host", default=None, help="foce a selenium hostname")
+
+
 @pytest.fixture(scope="session")
-def browser_name():
-    return os.environ["BROWSER"]
+def browser_name(pytestconfig):
+    return os.environ.get("BROWSER") or pytestconfig.getoption("--browser-name")
 
 
 @pytest.fixture(scope="session")
-def selenium_host(worker_id):
-    oktet = 1 if worker_id == "master" else int(worker_id.lstrip("gw")) + 1
-    host = f"127.0.0.{oktet}"
-    ps = subprocess.run(
-        [
-            "sudo",
-            "podman",
-            "run",
-            "--rm",
-            "-d",
-            "-p",
-            f"{host}:4444:4444",
-            "-p",
-            f"{host}:5999:5999",
-            "--shm-size=2g",
-            "quay.io/redhatqe/selenium-standalone:latest",
-        ],
-        stdout=subprocess.PIPE,
-    )
-    yield host
-    container_id = ps.stdout.decode("utf-8").strip()
-    subprocess.run(["sudo", "podman", "kill", container_id])
+def selenium_url(pytestconfig, worker_id):
+    forced_host = pytestconfig.getoption("--force-host")
+
+    if forced_host is None:
+        oktet = 1 if worker_id == "master" else int(worker_id.lstrip("gw")) + 1
+        host = f"127.0.0.{oktet}"
+        ps = subprocess.run(
+            [
+                "sudo",
```

Comment:
According to https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md Actions nodes should have `podman 2.0.4` that has fixed that issue https://github.com/containers/podman/issues/7016. You can try to remove it.

Reply:
that was art of debug, removed again, thanks for your hint